Git Testing/Scenes/Details/View/PictureDetailViewModel.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  PictureDetailViewModel.swift
+//  Git Testing
+//
+//  Created by BrainX Technologies on 19/01/2021.
+//
+
+import UIKit
+import Kingfisher
+
+class PictureDetailViewModel: NSObject {
+    //MARK:- PROPERTIES
+    private var view:UIViewController!
+    private var picture:Picture!{
+        didSet{
+            dataBindwithView(picture: picture)
+        }
+    }
+    
+    init(picture:Picture,viewController:UIViewController) {
+        self.picture = picture
+        self.view = viewController
+    }
+    //MARK: - BIND DATA
+    func dataBindwithView(picture: Picture!){
+        if let data = picture {
+            if let picture_url = URL( string: data.image_url ) {
+                (view as? PictureDetailVC)?.theImageView.kf.setImage(with: picture_url)
```

Comment:
In  my opinion viewmodel shouldn't be responsible to set the data in views. It should just provide simple apis to get the single unit of data to the view conntroller.